### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by amcdnl**
https://github.com/reaviz/reablocks/assets/1290843/d1d80665-51fb-4189-87e0-135e3c4db12a

### CODE REVIEW

File: src/form/Calendar/Calendar.tsx [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    } else {
+      if (!min) {
+        setViewValue(day);
+      } else if (max) {
+        setViewValue(day);
+      } else if (isBefore(min, day)) {
+        setViewValue(day);
+      } else {
+        setViewValue(day);
+      }
```
Comment (evgenoid) [inline src/form/Calendar/Calendar.tsx:148]: For what is this logic? And what does mean `viewValue`?

**Comment by evgenoid** [comment]
@amcdnl how does range should work?  Always select the range when `isRange` property is enabled or with some key holding like `shift`?

**Comment by amcdnl** [comment]
> What is the reason to highlight the same date in every month during switching?

Thats a bug

**Comment by amcdnl** [comment]
> 

If range is enabled, its a multi-select between 2 dates. No need for modifier keys.

**Comment by evgenoid** [comment]
@amcdnl could you take a look when you'll have a time?

**Pull request merged**